src/services/dragonchain-client/DragonchainClient.ts

```diff
@@ -1315,7 +1340,7 @@ export class DragonchainClient {
   /**
    * @hidden
    */
-  private generateQueryString = (queryObject: Map<string, string>) => `?${UrlSearchParams(queryObject)}`;
+  private generateQueryString = (queryObject: Map<string, string | number>) => `?${UrlSearchParams(queryObject)}`;
```

Comment:
instead of modifying the generate query string, it's safer for the functions calling this to cast any parameters to a string first, because all query params are rendered as strings, and you don't know how this is going to necessarily transform other data types

Reply:
Numbers are valid in query strings, so it makes sense for it to be able to take `string | number`. Most query string builders I've seen take either type--unless there is a fail case here that I'm not thinking of. 

> you don't know how this is going to necessarily transform other data types
The data types are known, string or number

Reply:
Numbers are valid in query strings because they are converted to a string, but I suppose this isn't a huge deal, probably just over concerned.